flycheck-pos-tip.el

```diff
@@ -82,16 +103,34 @@ Uses `pos-tip-show' under the hood."
   "Hide the Flycheck tooltip."
   (pos-tip-hide))
 
+(defvar flycheck-pos-tip-tty-old-message nil
+  "The contents of the message buffer before showing errors.")
```

Comment:
I think we don't need that.  The built-in error display of Flycheck doesn't bother to restore the former message either.